schema.graphql

```diff
@@ -230,7 +228,7 @@ type ANJTransfer @entity {
 
 type FeeMovement @entity {
   id: ID!
-  owner: Juror! # TODO: this should work for type Bytes
```

Comment:
I sometimes see similar comments but don't really understand the gap for closing the todos; were they schema problems or just things we didn't have time to try but thought would work?

Reply:
The problem we had was that we realized that not all fee movements necessarily correspond to jurors (e.g DAI is assigned to an appealer that won for a dispute that is not a juror). 

What we decided was to remove the FeeMovement <=> Juror association and do the queries filtering directly by the owner which is now an address.